### PR TITLE
Add a hook before dash.js media player initialization

### PR DESCRIPTION
**Pull request by gesinger**

### CODE REVIEW

File: src/js/videojs-dash.js [hunk 1]
```diff
@@ -48,6 +51,10 @@ var dashjs = require('dashjs');
       Html5DashJS.useVideoJSDebug(this.mediaPlayer_);
     }
 
+    if (Html5DashJS.beforeInitialize) {
+      Html5DashJS.beforeInitialize(player, this.mediaPlayer_);
+    }
```
Comment (forbesjo) [inline src/js/videojs-dash.js]: Can `beforeInitialize()` replace `useVideoJSDebug()`?

Comment (forbesjo) [inline src/js/videojs-dash.js]: Logging through videojs.log could then be used as an example.

Comment (gesinger) [inline src/js/videojs-dash.js]: Definitely, and good call. Will update.

File: README.md [hunk 2]
```diff
@@ -68,3 +68,19 @@ videojs.Html5DashJS.updateSourceData = function(source) {
...
 };
 ```
+
+## Before Initialize Hook
+
+Sometimes you may need to extend Dash.js, or have access to the Dash.js MediaPlayer before it is initialized. For these cases we have a beforeInitialize hook. The method is passed the Video.js player instance and the instance of Dash.js' MediaPlayer we are using, before the media player is initialized.
+
+```javascript
+videojs.Html5DashJS.beforeInitialize = function(player, mediaPlayer) {
+  // Log MedaPlayer messages through video.js
```
Comment (forbesjo) [inline README.md]: s/Meda/Media

**Comment by forbesjo** [comment]
Missing some test changes from https://github.com/videojs/videojs-contrib-dash/pull/75/files#diff-a7f35573a9a8c8e9ca9b2f511f994675R57

**Pull request merged**